Directory.Build.props

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="RoslynTools.Microsoft.RepoToolset" Version="$(RoslynToolsMicrosoftRepoToolsetVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.0-beta1-62314-01" />
```

Comment:
><PackageReference Inclu [](start = 4, length = 23)

This should probably go to src\Directory.Build.props - we don't need the analyzers in projects that build nuget packages.

Reply:
Done.

---
In reply to: [150717347](https://github.com/dotnet/roslyn-analyzers/pull/1356#discussion_r150717347) [](ancestors = 150717347)